resources/TrialApp.didact.md

```diff
@@ -0,0 +1,55 @@
+# Welcome to your Trial App Didact Tutorial
+
+This guide will show you how to make and deploy changes to your Azure App Service Trial App. Your trial will last 60 minutes. You can find more information about Azure App Service [here](https://docs.microsoft.com/en-us/azure/app-service/).
+
+## Getting Started
+
+### Step 1: Clone the trial app source code
+
+1. Locate your imported Trial App in the Azure view.
+
+<img src="./trialAppTutorial/viewTrialAppScreenshot.png" style="width: 256px;">
+
+2. Right click on the Trial App.
+3. Select `clone`.
+
+<img src="./trialAppTutorial/rightClickCloneScreenshot.png" style="width: 256px;">
+
+4. Choose a folder to clone the Trial App source code to.
+5. When the repository has finished cloning a notification like the one below will appear. Open the folder in VS Code or add it to the workspace.
+
+<img src="./trialAppTutorial/cloneNotificationScreenshot.png" style="width: 256px;">
+
+### Step 2: Make some changes to your app
+
+Make some changes to your trial app. Try changing some text in `template.html`.
+
+[Click here](didact://?commandId=workbench.view.explorer) to view the project files in the Explorer.
+
+### Step 3: Deploy your changes
+
+1. Select the blue up arrow icon to deploy your code to your Trial App.
```

Comment:
Personally, I think we should have the deploy scenario be one-click.  We can do that by leveraging the same settings that we use to set up one-click deploy for Azure users.

Reply:
Agreed. I like the idea of it being consistent, but if you can make it a one-click experience, go for it.

Reply:
It's now a one click experience for our happy path users 😄